scripting/tf2utils.sp

```diff
@@ -1000,6 +1036,36 @@ any Native_GetPlayerFromSharedAddress(Handle plugin, int numParams) {
 	return GetEntityFromAddress(pOuter);
 }
 
+any Native_StartLagCompensation(Handle plugin, int params) {
+	int client = GetNativeCell(1);
+	if(client < 1 || client > MaxClients || !IsClientInGame(client))
```

Comment:
Simplify this by using the `GetNativeInGameClient` stock function (which throws as needed).